aviatrix/resource_aviatrix_site2cloud_ca_cert_tag.go

```diff
@@ -0,0 +1,200 @@
+package aviatrix
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/AviatrixSystems/terraform-provider-aviatrix/v2/goaviatrix"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAviatrixSite2CloudCaCertTag() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAviatrixSite2CloudCaCertTagCreate,
```

Comment:
Should we use the context CRUD functions? Like `CreateContextWithoutTimeout`.

Reply:
Addressed comments.